src/suresh/maxLength.java

```diff
@@ -0,0 +1,41 @@
+package suresh;
+
+public class maxLength {
+
+	String[] arr = { "Harsh,", "Maulik", "Technocredits", "krisna" };
+
+	static String getmaxLength(String[] arr) {
+		int maxLength = 0;
+		String longestString = null;
+
+		for (String s : arr)
+
+			if (s.length() > maxLength) {
+				maxLength = s.length();
+				longestString = s;
+			}
+		return longestString;
+	}
+
+	public static void main(String[] args) {
+
+		maxLength maxlenth = new maxLength();
+		String[] arr = { "Harsh,", "Maulik", "Technocredits", "krisna" };
+
+		String longestString = getmaxLength(arr);
+
+		System.out.format("longest string: '%s'\n", longestString);
+
+		System.out.format("String length", longestString.length());
+
+		int count = 0;
+
+		for (int i = 0; i < longestString.length(); i++) {
```

Comment:
do you really need this Suresh ?? word has space then it is not a single word.

You didnt understand requirement correctly

display method to print all even charcter of the name having maximum length.
   void display(String name){
   
   }
   for technocredits output should be - ehordt 
   you missed to write that method.